src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -26,7 +26,8 @@
     ],
     "exclude": [
         "../../submodules/NuGet.Versioning/src/NuGet.Versioning/compiler/**/*.cs",
-        "../../submodules/**/AssemblyInfo.cs"
+        "../../submodules/**/AssemblyInfo.cs",
+        "../../submodules/Logging/src/Microsoft.Framework.Logging/LoggingServiceCollectionExtensions.cs"
```

Comment:
If we don't exclude this, we have to pull DI Interfaces into the AppHost, which we don't really want to do.